docs-source/content/security/service-accounts.md

```diff
@@ -33,5 +33,5 @@ look for the `serviceAccount` value using the Helm command:
 $ helm get values --all weblogic-operator
 ```
 #### Additional reading
-* [Helm service account]({{<relref "userguide/managing-operators/_index.md#install-helm-and-tiller">}})
+* [Helm service account]({{<relref "userguide/managing-operators/_index.md#install-helm">}})
```

Comment:
Did this cross-ref change? In the 3.0.1 docs, I see https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-operators/#install-helm-and-tiller

Reply:
there is no tiller now because only Helm V3 is supported.

Reply:
ok, so has someone changed this heading, Install Helm and Tiller?

Reply:
We probably need to remove #install-helm-and-tiller section completely.

Reply:
Someone has tried to remove references to tiller, but might have missed this one. The #install-helm-and-tiller section does not exist anymore.

Reply:
Thank you.

Reply:
yes, in the develop branch, but I did not see it removed in any of the 3.x releases.